client/state/site-settings/selectors.js

```diff
@@ -44,7 +44,7 @@ export function getSiteSettingsSaveRequestStatus( state, siteId ) {
  * @return {Object}        Site settings
  */
 export function getSiteSettings( state, siteId ) {
-	return get( state.siteSettings.items, [ siteId ], null );
+	return get( state, [ 'siteSettings', 'items', siteId ], null );
```

Comment:
Was this changed due to tests? Based on reducer shape of state we should always be able to assume that `items` exists. I think we'll want to update tests instead.